lib/trike/proxy.ex

```diff
@@ -90,6 +90,7 @@ defmodule Trike.Proxy do
       case Jason.encode(event) do
         {:ok, event_json} ->
           {:ok, _result} = state.put_record_fn.(stream, partition_key, event_json)
+          Logger.info(event_json)
```

Comment:
suggestion: it would be useful to have a label for this event, like
```suggestion
          Logger.info(["trike_ocs_event ", inspect(event_json)])
```
@paulswartz maybe there's a more clever way to do it with metadata instead?

Reply:
It's already encoded JSON, which Splunk can handle. So you want something like:
`Logger.info("trike_ocs_event json=#{event_json}")`

However, once #20 lands (hopefully Thursday AM) `event_json` will be a list, and I'm not sure that's what you want.